share/commands/server-trigger-policy-generation

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# @description trigger a policy generation
+# @man This command will trigger a policy generation, to generate
+# @man policies for any nodes that may have changed
+# @man +
+# @man *Options*:
+# @man +
+# @man *-i*: run the agent in information mode, displays all executed commands
+# @man +
+# @man *-c*: run the agent without color output
+
+. "${BASEDIR}/../lib/common.sh"
+. "${BASEDIR}/../lib/api_call.sh"
+
+TOKEN="$(cat /var/rudder/run/api-token)"
+DISPLAY_COMMAND=false
```

Comment:
This is never used, it should always be the 5th arg of filtered_api_call, see api_call.sh